### ISSUE TITLE
Offset - possible to target anchor or similar, rather than browser top?

### ISSUE THREAD

**Issue by JoshCarey** [WickyNilliams/headroom.js#129]
Basically I have a site I'm trying to develop that has a fixed header (non headroom related), a large hero graphic (which will change from page-to-page, and also because its responsive, the height in pixels is fluid), then a more vertically compact header that would be using headroom.

Layout is as follows:
(fixed header)
(hero image - different height depending on screen width - different images on different pages)
(slimmer header using headroom.js)

The functionality desired is that the user can scroll past the fixed header... because its fixed, the hero image (on a higher z-index) "slides up and covers" the header, then when reaching the bottom of the hero, the new header comes into view, then sliding out of view via the normal pixel offset.

So basically I'm looking to make the bottom edge of the hero image the "top" of where the offset would be based.

I hope that makes sense...  Is it possible to accomplish this somehow?
